ci/Jenkinsfile

```diff
@@ -2,6 +2,7 @@ node("docsbuilder-maven") {
   checkout scm
   stage("Build Documentation") {
     sh "pwd && ls -l"
+    sh "echo :revnumber: $(git rev-parse --short=10 HEAD) >> ./docs/topics/templates/document-attributes.adoc"
```

Comment:
Actually I think there's an environment variable that Jenkins exposes that contains the revision. That's better than `git rev-parse`, even though it probably isn't nicely shortened. Please hold this PR, I'll amend on Monday.